### ISSUE TITLE
Cannot Find Module ' WebPack '

### ISSUE THREAD

**Issue by SMWARREN** [hackmdio/hackmd#741]
![screen shot 2018-02-25 at 8 57 18 am](https://user-images.githubusercontent.com/5418757/36642214-2131971c-1a0a-11e8-870d-68dcad6baa3f.png)

The only way i can get this too build is too install each web-pack package separately using yarn which is a tedious. How can i fix these webpack errors when, i have webpack globally.

**Comment by SISheogorath** [comment]
Please run 

```
yarn install --pure-lockfile && \
yarn install --production=false --pure-lockfile
```

and run `yarn add webpack` additionally afterwards, when it still throws this error.

Once this is done, it should work.

**Comment by SMWARREN** [comment]
1. I ran the above code 
```
Please run

yarn install --pure-lockfile && \
yarn install --production=false --pure-lockfile
and run yarn add webpack additionally afterwards, when it still throws this error.

Once this is done, it should work.
```

![screen shot 2018-02-25 at 9 40 17 am](https://user-images.githubusercontent.com/5418757/36642643-8e616cd0-1a10-11e8-8a4c-a0f7f2175bbe.png)

2. I then ran yarn build. 
![screen shot 2018-02-25 at 9 40 29 am](https://user-images.githubusercontent.com/5418757/36642642-8e4fcf5c-1a10-11e8-9546-41fec51afc3a.png)

3. Then It asks me to install the extract plugin and when i do that i get caught in a loop which repeats 1-3.  I would have to run a yarn add webpack extract-text-webpack-plugin ... etc each webpack plugin
![screen shot 2018-02-25 at 9 40 50 am](https://user-images.githubusercontent.com/5418757/36642641-8e3f41a0-1a10-11e8-8735-5870606defe6.png)
